f1lger/AquaTrack-Frontend
# Fix images anastasiia

### **Description of Changes**
This PR includes the following updates:

- Updated images for the HomePage and Signup Page sections.

- Fixed the position of the logo on tablet and mobile versions.

- Resolved the redirection issue from the logo to the Home page.

### **What problem is being solved?**
Improved the user experience by ensuring correct image rendering and logo positioning across different devices.
Fixed the broken link that prevented proper redirection to the Home page when clicking the logo.

### **What are the main changes?**

- Image updates for key sections (HomePage and Signup Page).

- Adjustments to logo position for better display on tablets and mobile devices.

- Fix for logo redirection to the Home page.

### **Type of Changes**
🐛 Bug fix
🛠️ Code refactoring


### **Related Issues**
None

### **How to Test**

1. Visit the HomePage and Signup Page to ensure the correct images are displayed.
2. Test the logo's position on both tablet and mobile views to verify it is correctly aligned.
3. Click the logo on any page and confirm it redirects to the Home page.


### **Checklist**
